container/openejb-core/src/main/java/org/apache/openejb/assembler/classic/Assembler.java

```diff
@@ -3353,6 +3355,11 @@ public synchronized boolean containsKey(final Object key) {
                 classLoader = getClass().getClassLoader();
             }
 
+            /*
```

Comment:
Could the grammatical errors be the incorrect use of commas ?

Reply:
Please, what are the grammatical errors ?
Could the grammatical errors be the incorrect use of commas ?
The same text follows, with the correction of the use of commas, in the grammatical tense, of the pause in the text.

Below is the corrected text:
"The Import Sql class is deprecated, and should no longer be used, to import database objects. Use the Flyway or Liquibase tools for this purpose."

Or even: 

"The Import Sql class is deprecated, and no more is used, to import database objects. Use the Flyway or Liquibase tools for this purpose."

Reply:
> Please, what are the grammatical errors

I was referring to the last two sentences of the following comment: 

"The code below is outdated, because the ImportSql class, is no longer to be used. It is deprecated, be used to import database objects. Whether to use the Flyway or Liquibase tools."

I would just write: "The ImportSql class is deprecated and should no longer be used. Use Flyway or Liquibase instead." to be clear and concise.

Reply:
Ok